### PR TITLE
Fix inconsistent capitalization in main menu

### PR DESCRIPTION
**Pull request by wyatt400**
This fixes inconsistent capitalization in the main menu: "Create a New Profile" should be "Create a new profile", like "New private window with Tor"

Resolves brave/brave-browser#16503

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/handbook/blob/master/development/security.md#when-is-a-security-review-needed), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [x] A security review [is not needed](https://github.com/brave/handbook/blob/master/development/security.md#when-is-a-security-review-needed), or a link to one is included in the PR description
- [x] New files have MPL-2.0 license header
- [x] Adequate test coverage exists to prevent regressions
- [x] Major classes, functions and non-trivial code blocks are well-commented
- [x] Changes in component dependencies are properly reflected in `gn`
- [x] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Comment by bsclifton** [comment]
@wyatt400 can you please create an issue for this? Describing the problem in a way that reviewers and QA can see the issue and know how it would be fixed

**Comment by wyatt400** [comment]
@bsclifton Yes, here's the recently created issue at **brave/brave-browser**: https://github.com/brave/brave-browser/issues/16503

File: app/brave_generated_resources.grd [hunk 1]
```diff
@@ -176,10 +176,10 @@
         Wallet
       </message>
       <message name="IDS_ADD_NEW_PROFILE" desc="The app menu item to create a new User Profile">
-        Create a New Profile
+        Create a new profile
       </message>
       <message name="IDS_OPEN_GUEST_PROFILE" desc="The app menu item to create a new User Profile">
-        Open Guest Window
+        Open guest window
```
Comment (bsclifton) [inline app/brave_generated_resources.grd:182]: @rebron should this be `Open a guest window`?

**Comment by bsclifton** [comment]
Apologies for the crazy long review time @wyatt400 - I'll help get this reviewed in a timely manner now 😄 Just have one open question for @rebron otherwise it looks good

File: app/brave_generated_resources.grd (hunk 1 above)
Comment (bsclifton) [reply app/brave_generated_resources.grd:182]: checked with @rebron; leaving as-is works 😄

**Review by bsclifton — APPROVED** [review]
++

File: app/brave_generated_resources.grd [hunk 2]
```diff
@@ -167,7 +167,7 @@
         Brave Ad Block
       </message>
       <message name="IDS_SHOW_BRAVE_WEBCOMPAT_REPORTER" desc="The menu item to report a broken site in the app menu">
```
[the remark this answers is not in the document]
Comment (bsclifton) [reply app/brave_generated_resources.grd:169]: @wyatt400 even though I had already merged the code, you could do this as a follow up and we'll help you sort it out 😄

[the remark this answers is not in the document]
Comment (wyatt400) [reply app/brave_generated_resources.grd:169]: @bsclifton So is this being reverted?

[the remark this answers is not in the document]
Comment (bsclifton) [reply app/brave_generated_resources.grd:169]: No revert needed! You can create a new patch with the feedback above though 😄

**Pull request merged**